terratest/src/test/pega/data/values_hpa_disabletarget.yaml

```diff
@@ -10,4 +10,4 @@ global:
       hpa:
         enabled: true
         # enableCpuTarget should be true as default
-        enableMemoryTarget: false
+        # enableMemoryTarget should be false as default
```

Comment:
1. why is this editable if it should always be false?
2. move to line 13, before the parameter
3. change to->Ensure that enableMemoryTarget remains set to false.

Reply:
This file is not shipped, it's just a test asset.